HW_3/task_01.py

```diff
@@ -0,0 +1,11 @@
+#Дан список повторяющихся элементов.
+#Вернуть список с дублирующимися элементами.
+#В результирующем списке не должно быть дубликатов.
+
+my_list = [1, 6, 7, 2, 1, 4, 10, 22, 7, 6, 7]
+my_set = set()
+
+for item in my_list:
```

Comment:
Можно уменьшить количество итераций
```suggestion
for item in set(my_list):
```